content/docs/hello-world.md

```diff
@@ -15,7 +15,7 @@ ReactDOM.render(
 );
 ```
 
-Isso mostra um header dizendo "Hello, world!" na página.
+Isso mostra um header dizendo “Hello, world!” na página.
```

Comment:
Não entendi qual a diferença de uma aspas para outra, a minha aspas é assim: "

Reply:
@jhonmike coloquei desta forma para manter o padrão utilizado nas demais paginas. Com relação as diferenças, neste contexto não produz um resultado diferente no markdown. A forma tipografica ```"``` está definida no **_ISO-LATIN-1_** que posteriomente “caiu em deuso”, enquanto a forma ```“``` é uma pontuação geral definida no padrão unicode, não achei exatamente a origem desta tipografia, mas tem sido bastante adotada em diversos idiomas.  [Uma explicação interessante para este assunto encontra-se disponivel no livro “Unicode Demystified”, p.447](https://books.google.com.br/books?id=wn5sXG8bEAcC&pg=PA446&lpg=PA446&dq=mongolian+and+Basic+Latin+quotation+marks&source=bl&ots=J2bBr4PdXB&sig=ACfU3U3PuBxX7dB4VVnTSSc6qPG6IgIONQ&hl=pt-BR&sa=X&ved=2ahUKEwipqJafiMLhAhVWKLkGHaHfAr8Q6AEwDHoECAUQAQ#v=onepage&q=mongolian%20and%20Basic%20Latin%20quotation%20marks&f=false)

Reply:
legal, confesso que não sabia sobre o tema, mas se não interfere na formação do markdown não vejo problemas em dar merge neste PR!